### ISSUE TITLE
✨ Adding Wiki

### ISSUE THREAD

**Issue by N3v1** [TeamCodeArena/DeliverEase#130]
### Description

Hi @SID262000 @Devansh-Singh-Parmar,
I thought that it would be easier to have the wiki on this repo instead on mine. So I could need some help copying, rewriting and designing it.

- [ ] Copying the texts from my fork
- [ ] Rewrite it
- [ ] Design it

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by SID262000** [comment]
Please proceed and try winding up ASAP @N3v1, Many Thanks for all your inclusive ideas and brainstorming!

**Comment by Devansh-Singh-Parmar** [comment]
I'm in @SID262000 @N3v1 
Please assign.

**Comment by Devansh-Singh-Parmar** [comment]
@N3v1 Can you continue it? I am busy these days.

I did the homepage

**Comment by N3v1** [comment]
Certainly, @Devansh-Singh-Parmar, I can work on the remaining pages as well. Let me know how I can assist you further.

**Comment by N3v1** [comment]
I can work on it now @SID262000

**Comment by N3v1** [comment]
I‘ll be done tomorrow at 20:00 pm CET
